lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -183,19 +183,26 @@ public function listTableIndexes($table)
     /**
      * Returns true if all the given tables exist.
      *
-     * @param string[] $tableNames
+     * To avoid BC-break not all parameters are included.
+     *
+     * @param string|string[] $tableNames
+     * @param bool            $filtered   Optional parameter, if false ignores filters.
```

Comment:
@greg0ire I followed the discussion about this, and the BC break is indeed fixed, but the API is really really horrible.

Isn't it possible to declare a new method, instead of using dynamic args?

Also, patch should probably target `master`, since it's an addition (for `2.10`)

Reply:
Uh yeah sure I should have spotted this, even phpmd says booleans in signatures are a smell…

Reply:
What would be the appropriate method name? `tablesExistWithAllTable`?

Reply:
`tablesExistInUnfilteredSchema`?